editor/editor_zoom_widget.cpp

```diff
@@ -34,6 +34,12 @@
 #include "editor/editor_scale.h"
 #include "editor/editor_settings.h"
 
+// Min and Max are power of two in order to play nicely with successive increment.
+// That way, we can naturally reach a 100% zoom from boundaries.
+// (copied from "editor/plugins/canvas_item_editor_plugin.cpp")
+#define MIN_ZOOM 1. / 128
```

Comment:
```suggestion
#define MIN_ZOOM (1.0 / 128.0)
```

For the sake of making this macro safe from unexpected behavior, we should surround it with parenthesis.

Reply:
Good point, but I'll leave it for now, since it's (probably) temporary anyway.

Reply:
Can this change be made?

Reply:
@fire are you asking whether this is a valid change or whether I could commit that change? Because I'm not sure what you mean.

Reply:
> For the sake of making this macro safe from unexpected behavior, we should surround it with parenthesis.

Can you surround the code with parenthesis?

Reply:
Done.

Reply:
Remember to amend / squash.